### PR TITLE
Add note for s3 versioning

### PR DESCRIPTION
**Pull request by hashitop**
## Labels

- [ ] inaccuracy
- [X] clarification
- [ ] new docs
- [ ] cosmetic bug - fixing broken text or markup
- [ ] enhancement - changing the website's behavior/layout
- [ ] api - requires an update to the [changelog](https://www.terraform.io/docs/cloud/api/changelog.html)

### CODE REVIEW

**Comment by hashitop** [comment]
@justinretzolk can you help to review this? some users reported that the bucket size bloated due to the versioning was enabled where it is not actually required so I only added a little note there for clarity.

File: content/source/docs/enterprise/before-installing/reference-architecture/aws.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ stateless production installation.
 The following table provides high-level server guidelines. Of particular
 note is the strong recommendation to avoid non-fixed performance CPUs,
 or “Burstable CPU” in AWS terms, such as T-series instances.
-
+Object
```
Comment (justinretzolk) [inline content/source/docs/enterprise/before-installing/reference-architecture/aws.md:48]: I dunno if you intended to add this line

**Review by justinretzolk — DISMISSED** [review]
Other than the one line that seems to have been mistakenly added, this LGTM.

**Review by hashitop — COMMENTED** [review]
Removed the Object thingy

**Pull request merged**